product_docs/docs/biganimal/release/getting_started/managing_cluster.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: "Managing a cluster"
+description: Describes options for managing clusters
+---
+
+## Pausing and resuming clusters
+
+Pausing a cluster allows you to save on compute costs without losing data or cluster configuration settings. There is no additional cost for paused clusters, although you must still pay for any disk and object storage that the paused cluster requires. The ability to pause clusters gives you the flexibility to configure applications that don't separate storage and computing. 
+
+Pausing a cluster drops all existing connections, blocks future actions such as editing the cluster, and stops monitoring. While paused, you can't make or maintain connections, or read, write, or edit cluster settings. Automatic backups are paused. However, backups taken while the cluster was running remain until their retention period expires. 
+
+!!! Note
+    If you have defined the backup retention period for a cluster to be less that the duration of the pause, it is possible that write-ahead logs (WALs) may be deleted from the backup object store and resumption could fail.
+
+While paused, clusters are not upgraded or patched, but upgrades are applied when the cluster resumes. Pausing a high availability or Postgres Distributed cluster shuts down all cluster nodes. 
+
+After seven days, single node and high availability clusters automatically resume. Resuming a cluster applies any pending maintenance upgrades. Monitoring begins again.  
+
+### Pausing a cluster
+
+1.  Go to the [Clusters](https://portal.biganimal.com/clusters) page in the [BigAnimal portal](https://portal.biganimal.com). 
+2.  To pause the cluster, do one of the following:
+    -   In the row for the cluster, at right, select the ellipsis icon and then select **Pause Cluster**.
+    -   Select the cluster you want to pause. From **Quick Actions** on the cluster details page, select **Pause Cluster**.
+3.  Confirm that you want to pause the cluster. When the process finishes, Cluster Info shows the cluster has been paused.
+
+### Resuming a cluster
+
+1.  Go to the [Clusters](https://portal.biganimal.com/clusters) page in the [BigAnimal portal](https://portal.biganimal.com). 
+2.  To resume the cluster, do one of the following:
+    -   In the row for the cluster, at right, select the ellipsis icon and then select **Resume Cluster**.
+    -   Select the cluster you want to resume. From **Quick Actions** on the cluster details page, select **Resume Cluster**.
+3.  Confirm that you want to resume the cluster. When the process finishes, Cluster Info shows the cluster is in a healthy state.
```

Comment:
When the process finishes, the cluster status will show as Healthy.

also do you think we should mention that resuming the cluster may take a few minutes?

Reply:
Made both suggested changes